### ISSUE TITLE
Unable to reproduce results using the pre-trained models

### ISSUE THREAD

**Issue by reschu** [gengshan-y/expansion#4]
Thank you for sharing your great work!

I have a hard time reproducing the results reported in your repo/paper using the provided pre-trained models.
However, using the precomputed results, I can reproduce the respective error numbers.

I presume that the shared models in the repository and the models used for the paper, are not the same.

If so, could you please either
a) share precomputed results for the original model in the KITTI_2015 testing set, or
b) share the original pre-trained models?

That would be great help.
Thank you.

**Comment by gengshan-y** [comment]
Thanks for the question! The uploaded models are trained with a slightly different optical flow backbone. From my side, the uploaded models do produce slightly better numbers than the paper reported, but it is consistent with the numbers listed in the repo -- On KITTI val set, the MiD (1e-4) metric is 73.6 for the shared checkpoint, and 75.0 as reported in the paper.

Is this what you observe when running `bash run_eval.sh`?

**Comment by reschu** [comment]
Thanks for the quick reply.

This is not what I had observed in the first place. My initial results were much worse.
However, I could reproduce the numbers now.

The issue was a version mismatch in pytorch between the recommended (1.2.0) and the version I was using (1.6.0).
In particular, the default behavior of `torch.nn.functional.grid_sample` has changed:

> The default behavior up to version 1.2.0 was align_corners = True. Since then, the default behavior has been changed to align_corners = False, in order to bring it in line with the default for interpolate().

Downgrading pytorch to 1.2.0 should resolve my issue, or in my case, I explicitly passed `align_corners=True` in line 117 of `VCN_exp.py`.

The issue can be closed.
